sdk/cognitivelanguage/ai-language-text/swagger/README.md

```diff
@@ -550,3 +503,65 @@ directive:
     where: $.definitions.HealthcareAssertion
     transform: $.description = "An object that describes metadata about the healthcare entity such as whether it is hypothetical or conditional.";
 ```
+
+### Rename id and status in JobState
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.jobId["x-ms-client-name"] = "id";
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.status["x-ms-enum"].name = "OperationStatus";
+```
+
+## JS customizations
+
+### Update descriptions
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions..properties.stringIndexType
+    transform: >
+      $.description = "Specifies the measurement unit used to calculate the offset and length properties. For a list of possible values, see {@link KnownStringIndexType}.\n\nThe default is the JavaScript's default which is \"Utf16CodeUnit\".";
+```
+
+### Rename dates in JobState
+
+```yaml
+directive:
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.createdDateTime["x-ms-client-name"] = "createdOn";
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.expirationDateTime["x-ms-client-name"] = "expiresOn";
+  - from: swagger-document
+    where: $.definitions.JobState
+    transform: $.properties.lastUpdatedDateTime["x-ms-client-name"] = "modifiedOn";
+```
```

Comment:
These don't seem like JS-specific transforms.

Reply:
They use slightly different names as discussed in https://github.com/Azure/azure-sdk-for-js/pull/22298#discussion_r903150883. That being said, I moved them anyway to the general section and see how that goes.